bootstrap/Context/FailureContext.php

```diff
@@ -413,12 +412,13 @@ public function takeScreenshot($filename, ElementInterface $page, $driver)
         $content = null;
         $filename .= microtime(true);
         // If not selenium driver, extract the html and put in file.
-        if ($this->screenshotMode === self::SCREENSHOT_MODE_HTML || ! ($driver instanceof Selenium2Driver)) {
+        if ($this->screenshotMode === self::SCREENSHOT_MODE_HTML || ! ($driver instanceof DriverInterface)) {
```

Comment:
We can probably do it this way whilst adding a try catch method on the getScreenshot call for other drivers that don't actually support the method and fallback onto the html screenshot? Also this extension is covered by tdd and bdd (for bdd - not passing test but a visual test more so). Would be good to get those in as well.